language/move-prover/bytecode/tests/borrow/basic_test.exp

```diff
@@ -609,13 +609,13 @@ fun TestBorrow::test8(b: bool, n: u64, r_ref: TestBorrow::R): TestBorrow::R {
      var r2: TestBorrow::R
      var t_ref: &mut TestBorrow::R
      var $t6: bool
-     var $t7: TestBorrow::R
-     var $t8: &mut TestBorrow::R
-     var $t9: u64
+     var $t7: u64
```

Comment:
Those changes come from a bug fix in the reaching definitions processor. The Join operator was broken. Apparently the bug did not effect verification results, but was discovered while testing the new spec_instrumenter.